src/components/FolderPage/LinkInput/linkInputStyle.js

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components';
+import linkIcon from '../../../assets/icons/link.svg';
+
+export const LinkContainer = styled.article`
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  background-color: var(--background-color);
+  padding: 3.75rem 0 5.62rem 0;
+`;
+
+export const InputContainer = styled.div`
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+  height: 4.3rem;
+  max-width: 800px;
+  margin: 0 2rem;
+  border-radius: 0.9375rem;
+  border: 1px solid #6d6afe;
+  background: #fff;
+
+  @media (max-width: 768px) {
+    margin: 0 1rem;
+  }
+`;
+
+export const LinkInputStyle = styled.input`
```

Comment:
요렇게보면 input tag 같지 않은 네이밍이어서 컴포넌트명이 `-Input` 으로 끝나면 좋을 것 같아요.
```suggestion
export const LinkStyledInput = styled.input`
```